packages/inter-protocol/src/stakeFactory/stakeFactoryKit.js

```diff
@@ -296,9 +297,17 @@ const helperBehavior = {
       newDebt,
     });
 
-    stageDelta(clientSeat, vaultSeat, giveColl, wantColl, KW.Attestation);
-    stageDelta(clientSeat, vaultSeat, give, emptyDebt, KW.Debt);
-    manager.mintAndReallocate(toMint, fee, clientSeat, vaultSeat);
+    manager.mintAndReallocate(
+      clientSeat,
+      toMint,
+      fee,
+      harden([
+        [clientSeat, vaultSeat, { [KW.Attestation]: giveColl }],
+        [vaultSeat, clientSeat, { [KW.Attestation]: wantColl }],
+        [clientSeat, vaultSeat, { [KW.Debt]: give }],
```

Comment:
Could be
```suggestion
        [clientSeat, vaultSeat, { [KW.Attestation]: giveColl, [KW.Debt]: give }],
        [vaultSeat, clientSeat, { [KW.Attestation]: wantColl }],
```
Do you prefer the existing version?

Reply:
I tried that and the line wrapping made it less readable. I think this style make it more clear what's going on, coming from the stageDelta style.